substrate-node/pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -92,12 +111,39 @@ pub mod pallet {
     pub type FarmPayoutV2AddressByFarmID<T: Config> =
         StorageMap<_, Blake2_128Concat, u32, Vec<u8>, ValueQuery>;
 
+    // Input type for public config
+    pub type PubConfigIP4Input = IP<
+        BoundedVec<u8, ConstU32<{ pub_config::MAX_IP_LENGTH }>>,
+        BoundedVec<u8, ConstU32<{ pub_config::MAX_GATEWAY_LENGTH }>>,
+    >;
+    pub type PubConfigIP6Input = IP<
+        BoundedVec<u8, ConstU32<{ pub_config::MAX_IP6_LENGTH }>>,
+        BoundedVec<u8, ConstU32<{ pub_config::MAX_IP6_LENGTH }>>,
```

Comment:
Comparing to the above for IPv4, shouldn't this be MAX_IP6_GATEWAY_LENGTH or smthing?